library/modules/bigip_profile_persistence_universal.py

```diff
@@ -0,0 +1,609 @@
+#!/usr/bin/python
```

Comment:
you have not used module stubber to get this done, and i already can see a number of errors

Reply:
Ah... I forgot about the stubber completely. I'll use it for the next module I write for sure.